3.0.0/README.md

```diff
@@ -36,21 +37,116 @@ TileJSON manifest files use the JSON format as described in [RFC 8259](https://t
+
+#### 3.3.1 `id`
+
+REQUIRED.
+
+A string value representing the the layer id. For added context, this is referred to as the `name` of the layer in the [Mapbox Vector Tile spec](https://github.com/mapbox/vector-tile-spec/tree/master/2.1#41-layers).
+
+#### 3.3.2 `fields`
+
+REQUIRED.
+
+A JSON object whose keys and values are the names and descriptions of attributes available in this layer. Each value (description) MUST be a string that describes the underlying data. If no fields are present, the `fields` key MUST be an empty object.
+
+#### 3.3.3 `description`
+
+OPTIONAL.
+
+A string representing a human-readable description of the entire layer's contents.
+
+#### 3.3.4 `minzoom` and `maxzoom`
+
+OPTIONAL.
+
+A number representing the lowest/highest zoom level whose tiles this layer appears in. `minzoom` MUST be greater than or equal to the set of tiles' `minzoom`. `maxzoom` MUST be less than or equal to the set of tiles' `maxzoom`.
+
+These keys are used to describe the situation where different sets of vector layers appear in different zoom levels of the same set of tiles, for example in a case where a "minor roads" layer is only present at high zoom levels.
+
+```JSON
+{
+  "vector_layers": [
+    {
+      "id": "roads",
+      "description": "Roads and their attributes",
+      "minzoom": 2,
+      "maxzoom": 16,
+      "fields": {
+        "type": "One of: trunk, primary, secondary",
+        "lanes": "Number",
+        "name": "String",
+        "sidewalks": "Boolean"
+      }
+    },
+    {
+      "id": "countries",
+      "description": "Admin 0 (country) boundaries",
+      "minzoom": 0,
+      "maxzoom": 16,
+      "fields": {
+        "iso": "ISO 3166-1 Alpha-2 code",
+        "name": "English name of the country",
+        "name_ar": "Arabic name of the country"
+      }
+    },
+    {
+      "id": "buildings",
+      "description": "A layer with an empty fields object",
+      "fields": {}
+    }
+  ]
+}
+```
+
+## 3.4 `attribution`
 
 OPTIONAL. Default: null.
 
-Contains an attribution to be displayed when the map is shown to a user. Implementations MAY decide to treat this as HTML or literal text. For security reasons, make absolutely sure that this field can't be abused as a vector for XSS or beacon tracking.
+Contains an attribution to be displayed when the map is shown to a user. Implementations MAY decide to treat this as HTML or literal text. For security reasons, make absolutely sure that this content can't be abused as a vector for XSS or beacon tracking.
 
 ```JSON
 {
   "attribution": "<a href='http://openstreetmap.org'>OSM contributors</a>"
 }
 ```
 
-## 3.2 `bounds`
+## 3.5 `bounds`
 
 OPTIONAL. Default: [ -180, -85.05112877980659, 180, 85.0511287798066 ] (xyz-compliant tile bounds)
```

Comment:
Here (and in many other places) the type of the attribute is not explicitly mentioned. So:

OPTIONAL Array. Default[...]

Reply:
https://github.com/mapbox/tilejson-spec/pull/36/commits/5e75cf855792a11a4ecaf8d9ae3372dad03ac127